packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -434,6 +434,12 @@ export function getStylesDeclarations(
 			);
 		}
 
+		// For aspect ratio to work, other dimensions rules (and Cover block defaults) must be unset.
+		// This ensures that a fixed height does not override the aspect ratio.
+		if ( cssProperty === 'aspect-ratio' ) {
+			output.push( 'min-height: unset' );
+		}
+
```

Comment:
Same as above, do we need this here, given that we're already deleting min-height when setting aspect-ratio?

Reply:
Yes, if a block only has aspect-ratio set in global styles and not at the individual block level, we still need to unset any `min-height` rule that might be coming from the block's CSS (as is the case with the Cover block's default CSS)

Reply:
Oooh I see, this is intended to cover legacy min-height implementations. Thanks for explaining!